### PR TITLE
✨ Adding CAPN clusterctl

### PR DESCRIPTION
**Pull request by christopherhein**
**What this PR does / why we need it**:
This makes it possible to deploy the `v0.1.0` CAPN release.

Xref: https://github.com/kubernetes-sigs/cluster-api-provider-nested/issues/54

Signed-off-by: Chris Hein <me@chrishein.com>

### CODE REVIEW

**Comment by enxebre** [comment]
lgtm though I'm not very familiar with clusterctl yet. Is there any docs illustrating this new provider choice?

File: cmd/clusterctl/cmd/config_repositories_test.go [hunk 1]
```diff
@@ -46,6 +47,7 @@ func Test_runGetRepositories(t *testing.T) {
 			g.Expect(err).ToNot(HaveOccurred())
 
 			if val == RepositoriesOutputText {
+				fmt.Print(string(out))
```
Comment (fabriziopandini) [inline cmd/clusterctl/cmd/config_repositories_test.go:50]: Leftover?

**Review by fabriziopandini — COMMENTED** [review]
Great to see CAPN using clusterctl!
lgtm apart a small nit.

(eventually in follow up PR) you might also want to add CAPN to:
- https://cluster-api.sigs.k8s.io/user/quick-start.html
- https://cluster-api.sigs.k8s.io/reference/providers.html

**Comment by sbueringer** [comment]
Nice!
lgtm apart from @fabriziopandini 's finding

File: cmd/clusterctl/cmd/config_repositories_test.go (hunk 1 above)
Comment (christopherhein) [reply cmd/clusterctl/cmd/config_repositories_test.go:50]: Yeah, left over from testing. Thanks for catching that.

**Review by vincepri — APPROVED** [review]
/lgtm

**Comment by christopherhein** [comment]
@fabriziopandini & @vincepri updated and removed the `fmt` prints I'd accidentally left in the `_test.go`.

**Review by vincepri — APPROVED** [review]
/approve
/lgtm

**Pull request merged**